core/base/model/AliceSkill.py

```diff
@@ -137,26 +177,26 @@ def loadWidgets(self):
 				)
 
 
+	def getWidgetInstance(self, widgetName: str) -> Optional[Widget]:
+		return self._widgets.get(widgetName)
+
+
 	def getUtterancesByIntent(self, intent: Intent, forceLowerCase: bool = True, cleanSlots: bool = False) -> list:
-		utterances = list()
+		lang = self.LanguageManager.activeLanguage
+		if not lang in self._intentsDefinitions:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [test for membership should be 'not in'](https://app.codacy.com/app/ProjectAlice/ProjectAlice/pullRequest?prid=4699682)